<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>727</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Calculo de abras</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 237, 216);
background-color: rgb(218, 255, 194);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 24pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));
background-color: rgb(217, 217, 217);
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string> Calculos de Obras</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 18pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Nombre :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Dinero presupuestado:  $</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Porcentaje de mano de obra:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Porcentaje de materiales:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Porcentaje de dinero extra:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Nombre">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Nombre y apellidos</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Presupuesto">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="PorcentajeMano">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>%</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="PorcentajeExtra">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>%</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="PorcentajeMateriales">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Precio total de los materiales: $</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Numero de trabajadores:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Sueldo de los trabajadores: $</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Dinero extra gastado: $</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Sueldo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="DineroGastado">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Precio de la construcci√≥n: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotal">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);
font: 87 12pt &quot;Arial Black&quot;;
border-top-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFalSob">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>580</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);
font: 87 12pt &quot;Arial Black&quot;;
border-top-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGuardar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>640</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCalcular">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Calcular</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>640</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>640</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="PrecioMats">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Numero">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>#</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPresupuesto">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);
font: 87 12pt &quot;Arial Black&quot;;
border-top-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMano">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);
font: 87 12pt &quot;Arial Black&quot;;
border-top-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMats">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);
font: 87 12pt &quot;Arial Black&quot;;
border-top-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelExtra">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 212, 212);
font: 87 12pt &quot;Arial Black&quot;;
border-top-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Sumatoria de presupuestos: $</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/prefijoNuevo/4786.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <zorder>label_17</zorder>
  <zorder>label_13</zorder>
  <zorder>label_12</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>Nombre</zorder>
  <zorder>Presupuesto</zorder>
  <zorder>PorcentajeMano</zorder>
  <zorder>PorcentajeExtra</zorder>
  <zorder>PorcentajeMateriales</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>Sueldo</zorder>
  <zorder>DineroGastado</zorder>
  <zorder>label_11</zorder>
  <zorder>labelTotal</zorder>
  <zorder>pushButtonGuardar</zorder>
  <zorder>pushButtonCalcular</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>PrecioMats</zorder>
  <zorder>Numero</zorder>
  <zorder>labelExtra</zorder>
  <zorder>label_19</zorder>
  <zorder>labelPresupuesto</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>labelMats</zorder>
  <zorder>labelMano</zorder>
  <zorder>label_16</zorder>
  <zorder>labelFalSob</zorder>
 </widget>
 <tabstops>
  <tabstop>Nombre</tabstop>
  <tabstop>Presupuesto</tabstop>
  <tabstop>PorcentajeMano</tabstop>
  <tabstop>PorcentajeMateriales</tabstop>
  <tabstop>PorcentajeExtra</tabstop>
  <tabstop>Sueldo</tabstop>
  <tabstop>DineroGastado</tabstop>
  <tabstop>pushButtonCalcular</tabstop>
  <tabstop>pushButtonGuardar</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_7</tabstop>
 </tabstops>
 <resources>
  <include location="imagen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Nombre</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>663</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Presupuesto</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>PorcentajeMano</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>PorcentajeMateriales</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>PorcentajeExtra</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Sueldo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>653</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>DineroGastado</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>labelTotal</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>labelFalSob</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>629</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>PrecioMats</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Numero</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>labelMano</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>labelMats</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>labelExtra</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>labelPresupuesto</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
